frontend/dockerfile/docs/reference.md

```diff
@@ -2224,6 +2224,12 @@ RUN echo "I'm building for $TARGETPLATFORM"
 | `BUILDKIT_SYNTAX`                     | String | Set frontend image                                                                                                                                                                                             |
 | `SOURCE_DATE_EPOCH`                   | Int    | Set the UNIX timestamp for created image and layers. More info from [reproducible builds](https://reproducible-builds.org/docs/source-date-epoch/). Supported since Dockerfile 1.5, BuildKit 0.11              |
 
+> **Warning**
```

Comment:
Maybe `Deprecated` could work here as well. Wondering what looks more "scary" to people (as we want it to look scary)

Reply:
I prefer `Warning` as it shown a better callout when rendered on GitHub or on docs website:

![image](https://user-images.githubusercontent.com/1951866/216782945-11a544a9-4073-494e-9c65-86c26a30f189.png)

Not sure if we can have another title with the same effect.